DelvUI/Interface/HudHelper.cs

```diff
@@ -104,6 +105,14 @@ public bool IsElementHidden(HudElement element)
                 return ConfigurationManager.GetInstance().LockHUD;
             }
 
+            // hide in gold saucer
+            var results = GoldSaucerIDs.Where(id => id == Plugin.ClientState.TerritoryType);
+            bool hasZoneIdMatch = results.Count() > 0;
+            if (Config.HideInGoldSaucer && hasZoneIdMatch)
+            {
+                return true;
+            }
+
```

Comment:
Kinda nitpicking but you should only check the array if the config is enabled, not before.

Reply:
You can just do if (enabled && array.Where(blabla).Count > 0)

conditions get checked in order so if the first one fails the 2nd one shouldn't be executed

Reply:
well at first i tried:
if (Config.HideInGoldSaucer && GoldSaucerIDs.Where(id => id == Plugin.ClientState.TerritoryType).Count() > 0)

and the compiler complained about something so i changed it to the way you see above